bigbluebutton-html5/imports/startup/server/index.js

```diff
@@ -6,12 +6,24 @@ import './settings';
 import Logger from './logger';
 import Redis from './redis';
 
+var parse = Npm.require('url').parse;
+
 const AVAILABLE_LOCALES = fs.readdirSync('assets/app/locales');
+const CDN_URL = process.env.CDN_URL || '';
 
 Meteor.startup(() => {
   const APP_CONFIG = Meteor.settings.public.app;
   const env = Meteor.isDevelopment ? 'development' : 'production';
-  Logger.warn(`SERVER STARTED. ENV=${env}, nodejs version=${process.version}`, APP_CONFIG);
+
+  // Add CDN
+  WebAppInternals.setBundledJsCssPrefix(CDN_URL);
+
+  // Trust the URL in our browser policy (if it's available).
+  try {
+    return BrowserPolicy.content.allowOriginForAll(CDN_URL);
```

Comment:
Looking at examples of BrowserPolicy use, others seem to import it first https://forums.meteor.com/t/solved-browserpolicy-question-how-to-allow-stripe/45787

Reply:
Removed, the HTML5CLIENT is a consumer not a provider here.